lib/kubeclient/missing_kind_compatibility.rb

```diff
@@ -17,7 +17,7 @@ class MissingKindCompatibility
         'replicationcontrollers'     => 'ReplicationController',
         'resourcequotas'             => 'ResourceQuota',
         'secrets'                    => 'Secret',
-        'securitycontextconstraints' => 'SecurityContextConstraints',
+        'securitycontextconstraints' => 'SecurityContextConstraint',
```

Comment:
oh I see you wrote we define the same method twice. We need to mitigate it like we did for endpoints:
https://github.com/abonas/kubeclient/blob/1e7e4f4db3c3f79985651db298d60f6ca31be778/lib/kubeclient/common.rb#L132